recipe/meta.yaml

```diff
@@ -37,11 +37,10 @@ requirements:
     - libcurl 7.88.1
     - draco 1.5.6
     - geotiff 1.7.0
-    - hdf5 1.10.6              # [not (osx and arm64)]
-    - hdf5 1.12.1              # [osx and arm64]
+    - hdf5 1.12.1
```

Comment:
```suggestion
    - hdf5 {{ hdf5 }}
```

Reply:
I can't do that because CBC defines different versions depending on the OS. See https://github.com/AnacondaRecipes/aggregate/pull/295.

Reply:
You totally can. When building the osx-arm64 variants, hdf5 jinja variable will be set to 1.12.1, when building others it will be set to 1.10.6. Unless I am missing a point.

Reply:
hdf5 1.10.6 should also be compiled with openssl 3 then. Or if not possible, a rebuild of packages against hdf5 1.12.1 will be required.

Reply:
Rebuilding packages that depend on hdf5 should technically be simple. There are only 3 packages left to rebuild: `opencv`, `h5py` and `pytables`. Also, seeing how much packages depend on hdf5 (less than 10), I question the choice of having it in cbc at all. I have a feeling it was added to our cbc file to handle the version discrepancy between platforms.

Reply:
A few of them have high-profile packages depending on them. This is all to ensure compatibility.
As to why the version discrepancy between platform, that is from not doing the work of aligning versions when osx-arm64 was added.

If what's left to rebuild is not much, then that should be the option.

Reply:
I see other packages have been updated to 1.12.1. This can use 1.12.1 directly.

Reply:
I rebuilt `opencv` (https://github.com/AnacondaRecipes/opencv-feedstock/pull/27), `h5py` (https://github.com/AnacondaRecipes/h5py-feedstock/pull/12 and https://github.com/AnacondaRecipes/h5py-feedstock/pull/13) and `pytables` (https://github.com/AnacondaRecipes/pytables-feedstock/pull/6).